kit/inputs/src/UIMaterialTextView/UIMaterialTextView.tsx

```diff
@@ -285,11 +285,11 @@ const UIMaterialTextViewFloating = React.forwardRef<
     UIMaterialTextViewRef,
     UIMaterialTextViewCommonProps
 >(function UIMaterialTextViewFloatingForwarded(props: UIMaterialTextViewCommonProps, passedRef) {
-    const { label, onLayout, children, onHeightChange, ...rest } = props;
+    const { label, onLayout, children, onHeightChange, success, ...rest } = props;
     const ref = React.useRef<TextInput>(null);
     const {
         inputHasValue,
-        clear: clearInput,
+        clear: clearProp,
```

Comment:
Btw it's not a prop, it's not passed as a prop anywhere

Reply:
ah